mono/Makefile.am

```diff
@@ -1,3 +1,5 @@
+include $(top_srcdir)/mk/common.mk
```

Comment:
can we avoid a new top-level folder just for this file? 😄 
we'd need to add it to some EXTRA_DIST too otherwise it'll fail the Linux tarball build.

maybe we should just add the `MAKEFLAGS` setting into each file instead of the include?

Reply:
I would love to put more in this directory, like a better way of testing the different configurations we care about, by defining various `check-*` targets. This is a bigger RFC for sure, so we shouldn't block this PR for that. If nothing ends up coming out, we can just remove this directory and inline its content where necessary.